### ISSUE TITLE
ID 258554: [Supporting the platform - Simon - Link>New Link]: Horizontal scroll bar is appearing on the screen after setting the view port at 320*256.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66489]
### **User Experience:** 
Low-vision users will not be able to navigate through all the controls properly and will get challenges while navigating on the controls present on the page as they would need to scroll a lot due to two-dimensional scrolling. 
_Note: User credentials should NOT be included in the bug._

### **Pre-requisite:**  
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org/). Sign in with valid credentials. "Home" page appears. 
2. Navigate the Feature Showcase pane heading and select it. 
3. Navigate the expanded Dropdown and select the "Edit" menu item. 
4. Navigate the page "Edit Panel" page which appears. 
5. Navigate to the "Settings" button and select it. 
6. Navigate the "Links" tab item and select it. 
7. Navigate the "Links" tab and select "New Link" button. 
8. Navigate the "Links Edit" page which appears. Verify the issue.

**Actual Result:** 
A horizontal scroll bar appears on the screen after setting the view port at 320*256. There is no loss of any data on the page.

**Expected Result:**
A horizontal scroll bar should not appear on the screen after setting the view port at 320*256. Navigation should be done only in vertical scrolling in 320*256px.
_Note:  The same issue exists with the "Inspector" button's dialog present on the same page "Explore". Path: Explore>change value in Container combo box to "Prometheus">Query pane>Inspector> Error_

### **Severity:**
`SEV2`

### **Area:**
?

**Comment by JAYANTJOSHI001** [comment]
@polibb @eledobleefe can you assign this issue to me
